web/groups.go

```diff
@@ -107,6 +107,17 @@ func (s *AutograderService) updateGroup(ctx context.Context, sc scm.SCM, request
 		return err
 	}
 
+	// allow changing the name of the group only if the group
```

Comment:
Could this instead be converted to a method on Group or something? And maybe instead of reassigning the `group.Name` field, that method could return the new name.

Reply:
Well, it expects a set of the existing groups to check against. Do you suggest a method on Group that just takes a slice of groups and compares the names?